filebeat/module/haproxy/log/_meta/fields.yml

```diff
@@ -39,6 +36,46 @@
     - name: raw_request_line
       description: raw_request_line is the complete HTTP request line, including the method, request and HTTP version string.
       type: text
-  
+      
+    - name: time_active_ms
+      description: time_active_ms is the time the request remained active in haproxy, which is the total time in milliseconds elapsed between the first byte of the request was received and the last byte of response was sent.
+      type: long
 
+    - name: time_wait_without_data_ms
+      description: time_wait_without_data_ms is the total time in milliseconds spent waiting for the server to send a full HTTP response, not counting data.
+      type: long
 
+    - name: time_wait_ms
+      description: time_wait_ms is the total time in milliseconds spent waiting for a full HTTP request from the client (not counting body) after the first byte was received.
+      type: long
+  
+- name: default
+  description: Default HAProxy log format
+  type: group
+  fields:
+  - name: facility
+    type: text
+    description: One of the 24 standard syslog facilities. Refer to https://cbonte.github.io/haproxy-dconv/1.7/configuration.html#3.1 for more information
+  - name: priority
+    type: long
+    description: Self descriptive
+  - name: frontend_name
+    type: text
+    description: Name of the frontend (or listener) which received and processed the connection
+  - name: mode
+    type: text
+    description: The mode the frontend is operating (TCP or HTTP)
+  - name: logsource
+    type: text
+    description: The HAProxy source of the log
+
+- name: tcp
+  description: TCP log format
+  type: group
+  fields:
+  - name: processing_time_ms
+    type: long
+    description: Total time in milliseconds elapsed between the accept and the last close
+  - name: connection_waiting_time_ms
+    type: long
+    description: Total time in milliseconds elapsed between the accept and the last close
```

Comment:
Other fields in this file start with the name of the metric, we should probably open another PR to rephrase them for consistency, @sayden can you take care of this?